VideoLocker/src/main/java/org/edx/mobile/view/adapters/InfiniteScrollUtils.java

```diff
@@ -124,14 +125,31 @@ public void loadMore() {
         }
 
         private void onLoadMore() {
+            onLoadMore(false);
+        }
+
+        /**
+         * Variant of the {@link #onLoadMore} function allowing us to control the visibility of
+         * progress and lazily clear adapter after a page has loaded.
```

Comment:
Nitpick: this is the main method, not a variant :stuck_out_tongue: I suggest either remove the description, or have it be the same as the no-args one.